graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/AdminSchemaBuilder.java

```diff
@@ -242,6 +242,12 @@ public GraphQLSchema build() {
                   .description("The name of the namespace")
                   .type(nonNull(GraphQLString))
                   .build())
+          .argument(
+              newArgument()
+                  .name("version")
+                  .description(
+                      "The version of the schema to get. If not specified, it will return the latest version.")
+                  .type(GraphQLString))
```

Comment:
is it a proper way for defining an optional argument? (not using `nonNull`)?
Also, I wonder if the `GraphQLString` is the correct type. From what I see, there is no UUID scalar in grapqh-java. However, there is an open PR for it, see https://github.com/graphql-java/graphql-java-extended-scalars/pull/18/files. Should we wait for it? Then, we would be able to do:
```
public static GraphQLScalarType UUID = new UUIDScalar();
```

Reply:
Yes, that is the proper way to make it optional.

I think `String` is good enough: the version is an opaque value as far as users are concerned, they only read it and provide it back when they deploy a new one. No user should ever need to parse it as a UUID.